package.json

```diff
@@ -56,6 +56,7 @@
     "prop-types": "^15.5.8"
   },
   "devDependencies": {
+    "antd": "^3.12.2",
```

Comment:
Put Antd's Demo on CodePen ?   you can fork this link [react-drag-listview  for ant-design's table columns](https://codepen.io/raisezhang/pen/MoMoyz)  and change a suitable title!

Reply:
added